Add `osu!taiko World Cup 2025: Semifinals Recap` news post (#13153)
ppy/osu-wiki: news/2025/2025-04-17-osutaiko-world-cup-semifinals-recap.md

---
layout: post
title: "osu!taiko World Cup 2025: Semifinals Recap"
date: 2025-04-17 23:00:00 +0000
---

Intensity increases as we move further towards the pinnacle of this year's osu!taiko World Cup!

![](/wiki/shared/news/2025-04-17-osutaiko-world-cup-semifinals-recap/banner.jpg)

> The pressure was merciless in last week's Semifinals. Rising tempo and intensity with each and every matchup was guaranteed. With the skill gap narrowing down even more and with every player making their way through some the mapool's hardest challenges, we are delivering last weekend's highlights right to your doorstep!

Reviewer: ```suggestion
The pressure was merciless in last week's Semifinals. Rising tempo and intensity with each and every matchup was guaranteed. With the skill gap narrowing down even more and with every player making their way through some the mappool's hardest challenges, we are delivering last weekend's highlights right to your doorstep!
```

You enjoy those sweet stats? In that case you will definitely like all the intricate details of this round! Check them out in [one of our statistics spreadsheets](https://docs.google.com/spreadsheets/d/1NlBaWi0ZjPHRHPcNNDTvWrtHzKr61j7Dob3sB0IIJEk).

## Match highlights

### ::{ flag=US }:: United States vs. ::{ flag=CN }:: China

**[View the match information here!](https://osu.ppy.sh/community/matches/117778625)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=1h4m40s&video=2431479030" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Going into this losers' bracket match, the United States (#3) had the pickem votes on their side by a quite significant amount. This however meant for them, that they had to outperform their opponent China (#7), in order to make the people proud that believed in them. Them also being the higher seeded team, put extra pressure on the players of the US. But nobody could've expected what this match held for us.

> Early into the match, we knew that both teams are known for different skillsets. So we were expecting a match that was going to be very close. Starting strong, the United States decided to go with [P4koo - Hyperlight. (feat. Tsuyuri Karin)](https://osu.ppy.sh/beatmapsets/2351019#taiko/5059246) (NM2) for their first pick. Despite the conbo advange for China, the US won their pick by just 4,855 points due to having better accuracy. The game was on!

Reviewer: ```suggestion
Early into the match, we knew that both teams are known for different skillsets. So we were expecting a match that was going to be very close. Starting strong, the United States decided to go with [P4koo - Hyperlight. (feat. Tsuyuri Karin)](https://osu.ppy.sh/beatmapsets/2351019#taiko/5059246) (NM2) for their first pick. Despite the combo advantage for China, the US won their pick by just 4,855 points due to having better accuracy. The game was on!
```

Author: 2 am writing shenanigans...

It didn't take long for the United States to come back in full glory however. Winning the next 4 picks consecutively definitely helped putting extra pressure on China. First, the US picked [WyvernP - White Devil](https://osu.ppy.sh/beatmapsets/2194042#taiko/4641645) (NM3). Despite a really good score from ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), the US had the better accuracy across the board. China answered by picking the other Hidden map in the pool, namely [The Ghost Of 3.13 - Forgotten](https://osu.ppy.sh/beatmapsets/2351057#taiko/5059325) (HD2) thinking they can outperform the United States again. However, with 2 FCs on board for the US from ::{ flag=US }:: [Miniature Lamp](https://osu.ppy.sh/users/9821194) and ::{ flag=US }:: [R J](https://osu.ppy.sh/users/6490509) the United States won their first breakpoint, tying the score to 3-3.

> Brining the score to a stunning 5-3 for the United States, there was only one last map to win for them to advance into the Finals round. But China wasn't done yet. Winning [tn-shi - Annihilation in F# Minor](https://osu.ppy.sh/beatmapsets/2351018#taiko/5059245) (HR2) and [aoi08 - j9fqo80fdgcr](https://osu.ppy.sh/beatmapsets/2350992#taiko/5059144) (NM4) tied the score back again, with an incredible score from ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319) taking this already so intense match to tiebreaker.

Reviewer: ```suggestion
Bringing the score to a stunning 5-3 for the United States, there was only one last map to win for them to advance into the Finals round. But China wasn't done yet. Winning [tn-shi - Annihilation in F# Minor](https://osu.ppy.sh/beatmapsets/2351018#taiko/5059245) (HR2) and [aoi08 - j9fqo80fdgcr](https://osu.ppy.sh/beatmapsets/2350992#taiko/5059144) (NM4) tied the score back again, with an incredible score from ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319) taking this already so intense match to tiebreaker.
```

</div>

The Semifinals of this year's osu!taiko World Cup closed out with an incredible match between two unexpected titans. Singapore (#4) returned to the Taiko World Cup after taking a break over 2024, blowing everyone's expectations out of the water with their 4th place seeding. Similarly, United Kingdom (#10) have been surpassing expectations despite a very different roster of players that led them to their 2nd place result last year. This left everyone asking "which of these two underdogs will be making their appearance in the finals?".

The UK kicked off the match strong, warm-up checking Singapore by picking the speedy [WyvernP - White Devil](https://osu.ppy.sh/beatmapsets/2194042#taiko/4641645) (NM3) and securing a comfortable first point. Singapore then opened up the Free Mod pool, using their starting pick on [Rabbit House & Evin a’k - optimal solution](https://osu.ppy.sh/beatmapsets/2321605#taiko/4973868) (FM1) and taking a decisive win despite no mod advantage.

> Sticking with the No Mod pool, team United Kingdom confidently responded by picking [P4koo - Hyperlight. (feat. Tsuyuri Karin)](https://osu.ppy.sh/beatmapsets/2351019#taiko/5059246) (NM2). Unfortunately for them, this played right into Singapore's hands as they swiftly took a breakpoint into their favour. Singapore then further cemented their lead with an outstanding set of scores on [cluli - they besties](https://osu.ppy.sh/beatmapsets/2350993#taiko/5059153) (HD1). Aiming to ensure they did not give away another breakpoint, the new osu! original, [Kaitendaentai - Aether](https://osu.ppy.sh/beatmapsets/2351002#taiko/5059170) (HR1) was picked by the United Kingdom and expertly defended with a near three-way full combo and significant score increase from their earlier match against team Malaysia. Singapore continued to showcase their dominance in the FreeMod pool with a huge upscore of their own on [Tonari no Niwa wa Aoi (NIWASHI+Aoi) - Polacro](https://osu.ppy.sh/beatmapsets/2140365#taiko/4504718) (FM2). ::{ flag=SG }:: [awdse22](https://osu.ppy.sh/users/8743513) surprised the world with a HDHR full combo worth 1,097,143 points and the second best score of the weekend!

Reviewer: ```suggestion
Sticking with the No Mod pool, team United Kingdom confidently responded by picking [P4koo - Hyperlight. (feat. Tsuyuri Karin)](https://osu.ppy.sh/beatmapsets/2351019#taiko/5059246) (NM2). Unfortunately for them, this played right into Singapore's hands as they swiftly took a breakpoint into their favour. Singapore then further cemented their lead with an outstanding set of scores on [cluli - they besties](https://osu.ppy.sh/beatmapsets/2350993#taiko/5059153) (HD1). Aiming to ensure they did not give away another breakpoint, the new osu! original, [Kaitendaentai - Aether](https://osu.ppy.sh/beatmapsets/2351002#taiko/5059170) (HR1) was picked by the United Kingdom and expertly defended with a near three-way full combo and significant score increase from their earlier match against team Malaysia. Singapore continued to showcase their dominance in the Free Mod pool with a huge upscore of their own on [Tonari no Niwa wa Aoi (NIWASHI+Aoi) - Polacro](https://osu.ppy.sh/beatmapsets/2140365#taiko/4504718) (FM2). ::{ flag=SG }:: [awdse22](https://osu.ppy.sh/users/8743513) surprised the world with a HDHR full combo worth 1,097,143 points and the second best score of the weekend!
```
